harness/determined/common/experimental/experiment.py

```diff
@@ -131,7 +130,7 @@ def pause(self) -> None:
     def unarchive(self) -> None:
         self._session.post(f"/api/v1/experiments/{self.id}/unarchive")
 
-    def wait(self, interval: int = 5) -> ExperimentState:
+    def wait(self, interval: float = 0.5) -> ExperimentState:
```

Comment:
what's the reasoning behind this?

Reply:
oops, I meant to change it to a `float`, not to change the default value